### PR TITLE
Added guidelines for asking for help

### PR DESCRIPTION
**Pull request by squigjess**
- When a new help menu is created with `=help_menu`, a message containing guidelines on how to appropriately and effectively ask for help is included as well as the buttons.
- The guidelines are placed above the buttons, forcing the user to read (or at least scroll past) the guidelines before opening a new help thread.
- Compartmentalised the contents of the help message into a .txt file to allow for easier editing in the future if any guideline changes are needed down the line.
- Given that Discord has a 2000-character limit on messages sent by users _and_ bots, `split_txtfile()` will split the text file into adequately-sized chunks that can then be sent in multiple messages, one after the other.

### CODE REVIEW

**Review by ooliver1 — APPROVED** [review]

**Comment by squigjess** [comment]
Need to make some changes, please don't merge yet.

**Review by Skelmis — COMMENTED** [review]
Lgtm once you change it over to embeds

**Review by VincentRPS — APPROVED** [review]
lgtm

**Comment by squigjess** [comment]
Added the following to the closing msgs for both the Close button and `=close` command.
image

I'll make another PR later on to move the thread-closing code to a function once this has been merged.

File: cogs/help.py
Comment (Skelmis) [inline cogs/help.py:144]: Make this multi line, lets keep character line counts reasonable.

File: cogs/help.py
Comment (Skelmis) [inline cogs/help.py:219]: Make this multi line, lets keep character line counts reasonable.

**Review by Skelmis — CHANGES_REQUESTED** [review]

**Review by VincentRPS — APPROVED** [review]
lgtm

**Review by Skelmis — APPROVED** [review]

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
Imo the rules can be shortened a ton

**Review by TAG-Epic — APPROVED** [review]

**Pull request merged**
